pyproject.toml

```diff
@@ -12,7 +12,7 @@ build-backend = "setuptools.build_meta"
 [tool.cibuildwheel]
 manylinux-x86_64-image = "manylinux2014"
 manylinux-i686-image = "manylinux2014"
-skip = "pp* cp36* cp37* cp38* *musllinux*"
+skip = "pp* cp36* cp37* cp38* *musllinux* cp??t-*"
```

Comment:
I think we'd need `cp???t-*` really, I think?  The first valid match is `cp313t-*`, presumably.

Reply:
Thanks, yes, I meant `cp3??t-*`. I was trying to copy the suggestion in the documentation [here](https://cibuildwheel.pypa.io/en/stable/options/#build-skip-examples).